back-end/src/lms/views/students.py

```diff
@@ -196,9 +315,9 @@ def list(self, request: Request, *args, **kwargs) -> Response:
         user = request.user
         milgroup = None
 
-        if hasattr(user, "teacher"):
+        if hasattr(user, 'teacher'):
```

Comment:
:(

Reply:
Most of the strings use a single quote, so the linter thing does not allow double ones. I could use double quotes everywhere, but i prefer single ones 😈

Reply:
:((